### DISCUSSION TITLE
Importing other Data

### DISCUSSION THREAD

**Discussion by tonygem2022** [iTwin/itwinjs-core#3865] [Q&A]
Hello.

We are looking to import a PDMS model and Data plus potentially other 3rd party packages.

Using the iTwin model hub does not provide the flexibility required. We consider we need a .DGN format.

iTwin outlines the system is very interoperable with Aveva and Hexagon packages. Please point us to the correct method of importing this data that enables the capabilities to explore and utilise these external databases in more detail.

Also, please highlight the limitations of what effectively can be imported and what cannot be.

Thank-you. Tony

**Comment by pmconne** [comment]
@jffmarker or @abeesh can you provide information about to what extent synchronization of PDMS to iModel is supported?

**Reply by tonygem2022** [reply]
Thanks Paul. Perhaps synchronization in its true definition is not required. If a client simply provided a 3rd party package database like PDMS, we are only looking to Read and display, not write back by the definition we are used to.

Thanks Tony.

**Comment by jffmarker** [comment]
PDMS/E3D data is imported by using first an extractor and then a connector. Because PDMS and Smart 3D conversions need to run against the applications, they run as desktop applications and can be downloaded here: https://softwaredownloads.bentley.com/en/Home?Brand=iTwin

Documentation for running the Apps: https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/51124/conversions-to-itwin-and-plantsight-for-various-plant-applications-using-imodel-connectors

Additionally, if you have RVM + attribute files exported from PDMS/E3D, you could use the Navisworks Connector to sync the data.

**Reply by ovaisa** [reply]
We have been following the instructions from the document attached below for the installation of PlantSight E3D Bridge Extractor. Please look at the pictures below for reference. We do not see the PlantSight E3D Extractor Bridge in the "Product Name " drop down list.

![image](https://user-images.githubusercontent.com/83007711/177544097-aff8d7c8-9333-4201-bc3e-6d31c62a243e.png)

![image](https://user-images.githubusercontent.com/83007711/177544155-458702e1-7839-4016-ba8d-162a93830e8c.png)

Can you please advise what we might be doing wrong and how we can install the appropriate software

Thanks,

OvaisA

**Reply by timlawrence-bentley** [reply]
Hi Ovais,

The documentation was out of date as the names used in the licenses had changed. Please see the updated docs that can be found here:
https://communities.bentley.com/products/digital-twin-cloud-services/itwin-services/w/synchronization-wiki/51124/conversions-to-itwin-and-plantsight-for-various-plant-applications-using-imodel-connectors

Basically the names had the PlantSight removed from the names, so you are looking at add "E3D Bridge Extractor" and "Open 3D Model Connector".

Hope this helps,

Tim
